AI_ML/OBJECT_DETECTION/OBJECT_DETECTION/OBJECT_DETECTION.py

```diff
@@ -9,21 +9,22 @@
 
 @flojoy
 def OBJECT_DETECTION(default: Image) -> Image:
-    """The OBJECT_DETECTION node detects objects in the input image,
-    and returns an 'image' DataContainer with those objects highlighted.
+    """
+    The OBJECT_DETECTION node detects objects in the input image, and returns an 'image' DataContainer with those objects highlighted.
 
-    Parameters:
+    Parameters
     ----------
     None
 
-    Supported DC types:
-    -------------------
+    Supported DC types
```

Comment:
Not an allowed section, make this an "Inputs" section following the format `name : type` instead.